src/math/polysat/viable.cpp

```diff
@@ -742,72 +778,183 @@ namespace {
         while (e != first);
         return true;
     }
-    
-    bool viable::quick_bit_check(pvar v) {
-#if 0
-        return true;
-#endif
+
```

Comment:
must admit I am not tuned into what this should do. Possibly the trickier part of the pull request.

Reply:
We look for the smallest x with x >= bound such that it matches the bit-pattern